.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: ruby
 cache: bundler
 rvm:
-  - 2.2.7
-  - 2.3.4
-  - 2.4.1
+  - 2.3.6
+  - 2.4.3
+  - 2.5.0
```

Comment:
My impression was that we were waiting until 2.5.1 was out before we looked at 2.5....

Reply:
Hopper is already on 2.5.0